src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectFactory.cs

```diff
@@ -44,12 +50,18 @@ public VisualStudioProject CreateAndAddToWorkspace(string projectSystemName, str
             => CreateAndAddToWorkspace(projectSystemName, language, new VisualStudioProjectCreationInfo());
 
         public VisualStudioProject CreateAndAddToWorkspace(string projectSystemName, string language, VisualStudioProjectCreationInfo creationInfo)
```

Comment:
Can we [Obsolete] these as well? There's going to be uses of these in the C# and VB legacy project system shims, but I think moving the JTF.Run() to those then means it's hopefully more clear that those are the exception and nobody else should be calling these.

(I'm not sure how much we have third parties calling this via IVT...otherwise I'd just say delete them.)

Reply:
done.